### PR TITLE
Adds an overview for the Web MIDI API

### PR DESCRIPTION
**Pull request by rachelandrew**
The missing overview page for Web MIDI, I included some examples from the spec.

Reviewer: @jpmedley

### CODE REVIEW

File: files/en-us/web/api/web_midi_api/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ <dt>{{domxref("MIDIOutputMap")}}</dt>
+ <dd>Represents all of the available MIDI output ports.</dd>
+ <dt>{{domxref("MIDIAccess")}}</dt>
+ <dd>Provides the methods to list input and output devices, and to access an individual device.</dd>
+ <dt>{{domxref("MIDIPort")}}</dt>
+ <dd>Represents an individual MIDI port.</dd>
+ <dt>{{domxref("MIDIInput")}}</dt>
+ <dd>Provides a method for dealing with MIDI messages from an input port.</dd>
+ <dt>{{domxref("MIDIOutput")}}</dt>
+ <dd>Queues messages to the linked MIDI port, messages can be sent immediately or after a specified delay.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:31]:
```suggestion
 <dd>Queues messages to the linked MIDI port. Messages can be sent immediately or after a specified delay.</dd>
```

File: files/en-us/web/api/web_midi_api/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ <dt>{{domxref("MIDIMessageEvent")}}</dt>
+ <dd>Represents the event passed to {{domxref("MIDIInput.onmidimessage")}}.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:33]:
```suggestion
 <dd>The event passed to {{domxref("MIDIInput.onmidimessage")}}.</dd>
```

File: files/en-us/web/api/web_midi_api/index.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ <dt>{{domxref("MIDIConnectionEvent")}}</dt>
+ <dd>Represents the event passed to the {{domxref("MIDIAccess.onstatechange")}} and {{domxref("MIDIPort.onstatechange")}} event handlers, when a port becomes available or unavailable.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:35]:
```suggestion
 <dd>The event passed to the {{domxref("MIDIAccess.onstatechange")}} and {{domxref("MIDIPort.onstatechange")}} event handlers, when a port becomes available or unavailable.</dd>
```

File: files/en-us/web/api/web_midi_api/index.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+</dl>
+
+<h2>Examples</h2>
+
+<h3>Gaining access to the MISI port</h3>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:40]:
```suggestion
<h3>Gaining access to the MIDI port</h3>
```

File: files/en-us/web/api/web_midi_api/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+function onMIDIFailure(msg) {
+  console.log( "Failed to get MIDI access - " + msg );
+}
+
+navigator.requestMIDIAccess().then( onMIDISuccess, onMIDIFailure );</pre>
+
+<h3>Listing inputs and outputs</h3>
+
+<p>In this example the list or input and output ports are retrieved and printed to the console.</p>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:58]:
```suggestion
<p>In this example the list of input and output ports are retrieved and printed to the console.</p>
```

File: files/en-us/web/api/web_midi_api/index.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    <th scope="col">Status</th>
+    <th scope="col">Comment</th>
+   </tr>
+   <tr>
+    <td>{{SpecName('WebMIDI API','#midioutput-interface')}}</td>
+    <td>{{Spec2('WebMIDI API')}}</td>
+    <td>Initial definition.</td>
+   </tr>
+  </tbody>
+ </table>
```
Comment (jpmedley) [inline files/en-us/web/api/web_midi_api/index.html:108]: <p>{{Specifications}}</p>

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/web_midi_api/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
+---
+title: Web MIDI API
+slug: Web/API/Web_MIDI_API
+tags:
+  - API
+  - MIDI
+  - Overview
+  - Reference
+  - Web MIDI API
```
[the remark this answers is not in the document]
Comment (rachelandrew) [reply files/en-us/web/api/web_midi_api/index.html:9]: I guess we wait until this is resolved https://github.com/mdn/content/discussions/4920

[the remark this answers is not in the document]
Comment (rachelandrew) [reply files/en-us/web/api/web_midi_api/index.html:9]: and for now omit it, I can always do a roundup of overview pages once that is resolved.

[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/web_midi_api/index.html:9]: SGTM

**Review by jpmedley — APPROVED** [review]

**Pull request merged**